app/models/spree/price_decorator.rb

```diff
@@ -1,7 +1,14 @@
+require 'spree/localized_number'
+
 module Spree
   Price.class_eval do
     after_save :refresh_products_cache
 
+    # this method override can be removed when updating to Spree 2.3
```

Comment:
Can you use the `# TODO:` tag? This way it'll be easier to find when the time comes

Reply:
sure! there it is